src/rf/js/src/core/uploads.js

```diff
@@ -0,0 +1,35 @@
+'use strict';
+
+var _ = require('underscore'),
+    Evaporate = require('evaporate'),
+    coreSettings = require('../core/settings');
+
+var uploadFiles = function(files) {
+    var evap = new Evaporate({
+        signerUrl: coreSettings.get('signerUrl'),
+        aws_key: coreSettings.get('awsKey'),
+        bucket: coreSettings.get('awsBucket'),
+        logging: false
+    });
+    _.each(files, function(file) {
+        // TODO Later we'll want to store the id of the file upload. We can use
+        // it to cancel the upload if needed.
+        // var id = evap.add({ //
+        evap.add({
+            // TODO - NAMES ARE CURRENTLY JUST FOR TESTING.
+            // WE WANT TO USE A UUID FOR FILE NAMES.
```

Comment:
Agreed. I don't think there is any reason why we can't just discard incoming file names and just use UUIDs across the board.